File: W1/abcz3.c
#include <stdio.h>

int main() {
    int x, y, z;
    
    scanf("%d %d %d", &x, &y, &z);
    
    if ( x + y > z ) {
        printf("alpha\n");
    } else if ( x < y - z ) {
        printf("bravo\n");
    } else if ( y % z == 0 ) {
        printf("charlie\n");
    } else {
        printf("zulu\n");
    }
    
    return 0;    
}

File: PHP/abcz3.php
<?php
    $a = fgets(STDIN);
    $b = fgets(STDIN);
    $c = fgets(STDIN);
    $isZulu = true;
    
    if ( $a + $b > $c ) {
        echo "alpha" . PHP_EOL;
        $isZulu = false;
    }
    if ( $a < $b - $c ) {
        echo "bravo" . PHP_EOL;
        $isZulu = false;
    }
    if ( $b % $c == 0 ) {
        echo "charlie" . PHP_EOL;
        $isZulu = false;
    } 
    if ( $isZulu ) {
        echo "zulu" . PHP_EOL;
    }
?>
